### ISSUE TITLE
Show predictable device label

### ISSUE THREAD

**Issue by joseivanlopez** [openSUSE/agama#592]
Now there is a device selector that helps to select the correct device by showing predictable information (DASD bus id, model, etc). But, once the device is selected, the rest of the UI stills shows the kernel name (e.g., */dev/vda*).

The UI should show a more meaningul information about the selected device (e.g., model).

Moreover, the disk selector does not show the channel, WWPN and LUN for zFCP disks. And libstorage-ng does not report that info. But Agama could get the information from the zFCP disks exported at */org/opensuse/Agama/Storage1/zfcp_disks/* D-Bus tree.

**Comment by joseivanlopez** [comment]
Moreover, the disk selector does not show the channel id for zFCP disks. libstorage-ng only reports the channel id for the DASD devices. Agama could also use the information from the zFCP disks exported at */org/opensuse/Agama/Storage1/zfcp_disks/* tree.

**Comment by joseivanlopez** [comment]
Crazy idea for UI: tooltip (or similar) to show the complete information about the selected disk:

![Screenshot from 2023-07-05 12-17-09](https://github.com/openSUSE/agama/assets/1112304/868a89f3-091c-46b3-831c-ad58675b518e)

**Comment by shundhammer** [comment]
Summarizing what I just wrote on IRC:

That tooltip looks very good, in particular with those graphical elements, not just plain text. I like it.

But maybe we should add an "(i)" icon next to the hyperlink to make it more discoverable; the tooltip could open when clicking there _and_ when hovering over the hyperlink.

**Comment by dgdavid** [comment]
> Summarizing what I just wrote on IRC:
> 
> That tooltip looks very good, in particular with those graphical elements, not just plain text. I like it.
> 
> But maybe we should add an "(i)" icon next to the hyperlink to make it more discoverable; the tooltip could open when clicking there _and_ when hovering over the hyperlink.

+1

**Comment by dgdavid** [comment]
> > Summarizing what I just wrote on IRC:
> > That tooltip looks very good, in particular with those graphical elements, not just plain text. I like it.
> > But maybe we should add an "(i)" icon next to the hyperlink to make it more discoverable; the tooltip could open when clicking there _and_ when hovering over the hyperlink.
> 
> +1

After having a look to Popover and Tooltip PatternFly components and taking A11y into account, I have changed a bit my mind. Now, I think it should be a Popover triggered when the user clicks on a question mark _button_.

Tooltip is aimed for holding short contextual information while the Popover was designed for holding more elaborated data (although the recommendation is still short, 1 to 3 sentences at most), which is the case here. IIUC, the [recommendation](https://www.patternfly.org/components/popover/design-guidelines) is to show non-alert Popovers on click instead on hover. In any case, I'm more for triggering when the user clicks on the trigger button than when hovering it, which matches better with the behavior when the user is using the keyboard for navigation through the app.

In short, the link opens the selector. The question mark button shows more information about the selected device... and even we can (maybe should) add the link for opening the selector from there too.

**Comment by dgdavid** [comment]
> In short, the link opens the selector. The question mark button shows more information about the selected device... and even we can (maybe should) add the link for opening the selector from there too.

Although after a short chat with @joseivanlopez and thinking it twice... this _does not make sense_ at this moment and needs to be re-evaluated.

Basically, 

* For a single device selection, it's simpler to open the selector again to see the device details. Simpler and cleaner. no question marks, no tooltips, no popovers, no multiples ways to reach the selector.
* There is more complex use case, which is the case of multiple disk selection. Probably the approach should be the same, the user see "LVM over 3 devices" or so and the details are in the dialog for selecting disks, but this must be evaluated after https://github.com/openSUSE/agama/discussions/776

Thus, I'm closing this issue by now. We can open it again if needed after the other work is done.
